web/src/components/dashboard/panels/DashboardPanels.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <v-container>
+    <v-row>
+      <v-col>
+        <v-expansion-panels
+          rounded="lg"
+          bg-color="#efedf5"
+          :flat="true"
+          v-model="expanded"
+        >
+          <v-expansion-panel rounded="lg" :value="RESERVED_JUDGEMENT">
+            <v-expansion-panel-title>
+              <h4 class="px-2 py-2">Reserved judgements & decisions {{ reservedJudgementCount > 0 ? `(${reservedJudgementCount})` : '' }}</h4>
+            </v-expansion-panel-title>
+            <v-expansion-panel-text class="reserved-judgements-table">
+                <v-skeleton-loader class="my-1" type="table" :loading="judgementsLoading">
+                    <ReservedJudgementTable :data="reservedJudgements"/>
+                </v-skeleton-loader>
+            </v-expansion-panel-text>
+          </v-expansion-panel>
+        </v-expansion-panels>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script setup lang="ts">
+  import ReservedJudgementTable from '@/components/dashboard/panels/reserved-judgements/ReservedJudgementTable.vue';
+  import ReservedJudgementService from '@/services/ReservedJudgementService';
+  import { ReservedJudgement } from '@/types/ReservedJudgement';
+  import { computed, inject, ref, watch } from 'vue';
+
+  const expanded = ref([]);
+  const judgementsLoading = ref(false);
+  const reservedJudgements = ref<ReservedJudgement[]>([]);
+  const RESERVED_JUDGEMENT = 'reserved-judgement';
+  const reservedJudgementCount = computed(() => reservedJudgements.value.length);
+  const reservedJudgementService = inject<ReservedJudgementService>(
+    'reservedJudgementService'
+  );
+  if (!reservedJudgementService) {
+    throw new Error('Service(s) is undefined.');
+  }
+
+  watch(expanded, (newVal: string) => {
+    if(newVal == RESERVED_JUDGEMENT) {
+        judgementsLoading.value = true;
+        reservedJudgementService
+          .get()
+          .then((data: ReservedJudgement[]) => {
+            reservedJudgements.value = data;
+            judgementsLoading.value = false;
```

Comment:
apologies if this is a pattern in use already in the application, but it makes sense to me to have this in a finally block - to avoid showing a loading placeholder if the request errors out.